src/vector/index.html

```diff
@@ -37,6 +37,13 @@
     <section id="matrixchat" style="height: 100%;"></section>
     <noscript>Sorry, Riot requires JavaScript to be enabled.</noscript> <!-- TODO: Translate this? -->
     <% for (var i=0; i < htmlWebpackPlugin.files.js.length; i++) {
+        if (_.endsWith(htmlWebpackPlugin.files.js[i], 'olm.js')) {
+           var array = htmlWebpackPlugin.files.js;
+           htmlWebpackPlugin.files.js.unshift(htmlWebpackPlugin.files.js[i]);
+           htmlWebpackPlugin.files.js.splice(i, 1);
+        }
+       }
+       for (var i=0; i < htmlWebpackPlugin.files.js.length; i++) {
```

Comment:
1. You're putting olm first? Why?
 2. The `array` variable is unused.
 3. Indents.

Reply:
1. Because in webpack 3 they get randomly ordered which causes olm to sometimes not load. The old script relied on the case that the array is always ordered the same way.

2 and 3 can I fix after the weekend

Reply:
ah ok. Yeah, I can see how that would break. Generally all the indents should be 4 spaces.

Reply:
I guess I will leave the html as is but fix the Js code in there

Reply:
Also, I wonder if we should be using `chunksSortMode` for the ordering (although given the docs don't tell you anything about what the different modes do, perhaps your solution is better...)